setup.py

```diff
@@ -89,7 +89,7 @@ def run_tests(self):
 setup(
     name='odl',
 
-    version='0.2.3',
+    version='0.2.4',
```

Comment:
This is not the only place. Make sure you get all of them (3 if I remember right).